.github/workflows/dispatch-build-bottle.yml

```
@@ -77,7 +77,7 @@ jobs:
           HOMEBREW_GITHUB_PACKAGES_TOKEN: ${{secrets.HOMEBREW_CORE_GITHUB_PACKAGES_TOKEN}}
         run: |
           cd ~/bottles
-          brew pr-upload --verbose --keep-old --no-commit --root-url="https://ghcr.io/v2/Homebrew/linuxbrew-core"
+          brew pr-upload --verbose --keep-old --no-commit --root-url="https://ghcr.io/v2/Linuxbrew/linuxbrew-core"
```

```suggestion
          brew pr-upload --verbose --keep-old --no-commit --root-url="https://ghcr.io/v2/linuxbrew/core"
```
Org needs to be lowercase.
We're dropping the `homebrew-` or `linuxbrew-` prefix from the tap name.
I prefer `docker://ghcr.io/linuxbrew/core` over `https://ghcr.io/v2/linuxbrew/core`. That may first require PR https://github.com/Homebrew/brew/pull/11037

---

Uppercase Homebrew seems to work over at homebrew-core?

---

https://github.com/Homebrew/brew/blob/6d0275ff574bc3958455f26e7561847d8195168c/Library/Homebrew/github_packages.rb#L98-L99

Homebrew/brew lower-cases it for us, but it'd be more clear to use the lower case letters. Docker explicitly disallows upper case letters in the image name. See https://docs.docker.com/engine/reference/commandline/tag/